docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -490,11 +490,24 @@ assigned a subnational location within their country. Here is the data
 for the `percent of simulants assigned to each subnational location by sex <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/ff08145109e1434669f08afe702ffc5e3d45a6c2/data_prep/sqlns_subnational/subnational_percents.csv>`_. Note that all 3 countries are included 
 in this csv file. 
 
+Since LBWSG is done nationally, there is a risk in assigning locations 
+at birth that they will be incorrect by 6 months of age. Subnational 
```

Comment:
I think it would be more clear to state that the "population-distribution of subnational locations" will be incorrect rather than the locations themselves in this sentence.